pg_search/src/index/writer/index.rs

```diff
@@ -57,7 +56,7 @@ impl SearchIndexWriter {
     ) -> Result<Self> {
         let (parallelism, memory_budget, wants_merge) = resources.resources();
 
-        let (req_sender, req_receiver) = crossbeam::channel::bounded(CHANNEL_QUEUE_LEN);
+        let (req_sender, req_receiver) = crossbeam::channel::bounded(1);
```

Comment:
Some analysis showed we'd never have more than 1 message queued up at a time (obvious, in hindsight), so this is just a little cleanup.